### PR TITLE
Components: Add layout and navbar components

### LINKED ISSUE #22: Create the Basic Page Layout: Search form
Every page will need some basic layout structure, so let's create the Layout component for that.

Layout component should have 2 sections:
 
- [ ] Content 
- [ ] Navigation bar

search return

Link to design is here: https://www.figma.com/file/z0UzctMydWQXgxeZ00AwqG/Forms?node-id=1889%3A28

### PR DESCRIPTION
**Pull request by tbergq**
closes #22

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "ios": "expo start --ios",
     "android": "expo start --android",
-    "dev": "BABEL_ENV=web next",
+    "dev": "yarn concurrently \"BABEL_ENV=web next\" \" yarn server\" ",
```
Comment (FilipMessa) [inline package.json]: Can you pls create also command just for web?
Sometimes I would like to run these commands separately thanks.

File: packages/relay/src/Environment.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 // @flow
 
-import { GRAPHQL_URL } from 'react-native-dotenv';
+// import { GRAPHQL_URL } from 'react-native-dotenv';
 import { createEnvironment, createNetworkFetcher } from '@mrtnzlml/relay';
 
 export default createEnvironment({
-  fetcherFn: createNetworkFetcher(GRAPHQL_URL),
+  fetcherFn: createNetworkFetcher('http://localhost:4000'),
```
Comment (FilipMessa) [inline packages/relay/src/Environment.js]: some problem with `.env`?

Comment (tbergq) [reply packages/relay/src/Environment.js]: Yeah, I was not supposed to commit that. I will remove it

File: package.json (hunk 1 above)
Comment (tbergq) [reply package.json]: sure

**Comment by tbergq** [comment]
@FilipMessa 🚀

**Comment by tbergq** [comment]
@FilipMessa rebased 🚀

**Pull request merged**